core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -423,6 +491,33 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def defaultBillingProjectPolicies = {
+    Map(
+      SamBillingProjectPolicyNames.owner -> SamPolicy(Set(WorkbenchEmail(userInfo.userEmail.value)), Set.empty, Set(SamProjectRoles.owner)),
+      SamBillingProjectPolicyNames.workspaceCreator -> SamPolicy(Set.empty, Set.empty, Set(SamProjectRoles.workspaceCreator))
+    )
+  }
+
+  private def createBillingProjectInternal(createProjectRequest: CreateRawlsBillingProjectFullRequest): Future[PerRequestMessage] = {
+    for {
+      maybeProject <- dataSource.inTransaction { dataAccess =>
+        dataAccess.rawlsBillingProjectQuery.load(createProjectRequest.projectName)
+      }
+      _ <- maybeProject match {
+        case Some(_) => Future.failed(new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.Conflict, "project by that name already exists")))
+        case None => Future.successful(())
+      }
+
+      _ <- samDAO.createResourceFull(SamResourceTypeNames.billingProject, createProjectRequest.projectName.value, defaultBillingProjectPolicies, Set.empty, userInfo, None)
+
+      _ <- dataSource.inTransaction { dataAccess =>
+        dataAccess.rawlsBillingProjectQuery.create(RawlsBillingProject(createProjectRequest.projectName, CreationStatuses.Ready, Option(createProjectRequest.billingAccount), None, None, createProjectRequest.servicePerimeter))
```

Comment:
it is ready because for v2 workspaces we create google-projects at the workspace level, not this (the billing-project) level. So there is not google-project in this code path so nothing to set a parent to.

Reply:
right! after posting the question I remembered that it's "project per workspace" 😅